src/auto/device.rs

```diff
@@ -206,9 +207,11 @@ impl Device {
         }
     }
 
-    //pub fn list_axes(&self) -> /*Ignored*/Vec<Atom> {
-    //    unsafe { TODO: call ffi::gdk_device_list_axes() }
-    //}
+    pub fn list_axes(&self) -> Vec<Atom> {
+        unsafe {
+            FromGlibPtrContainer::from_glib_container(ffi::gdk_device_list_axes(self.to_glib_none().0))
```

Comment:
Indeed, I'm also getting 0's for `list_axes` for my two devices. Do you know what a `GdkDevice` is in practice, and output device or an input device? I can't glean much information from various internet searches.